Sources/Carting/Commands/Update.swift

```diff
@@ -14,11 +14,20 @@ struct Update: ParsableCommand {
     var options: Options
 
     func run() throws {
-        let projectService = try ProjectService(projectDirectoryPath: options.path,
-                                                frameworksDirectoryPaths: options.frameworksDirectoryPaths)
-        try projectService.updateScript(withName: options.script,
-                                        format: options.format,
-                                        targetName: options.target,
-                                        projectNames: options.projectNames)
+        let projectService = ProjectService(projectDirectoryPath: options.path)
+        if !options.projectNames.isEmpty,
+           let frameworksDirectoryName = options.frameworksDirectoryNames.first {
+            try projectService.updateScript(withName: options.script,
+                                            format: options.format,
+                                            targetName: options.target,
+                                            projectNames: options.projectNames,
+                                            frameworksDirectoryName: frameworksDirectoryName)
+        }
+        else {
+            try projectService.updateScript(withName: options.script,
+                                            format: options.format,
+                                            targetName: options.target,
+                                            frameworksDirectoryNames: options.frameworksDirectoryNames)
+        }
```

Comment:
This code is identical to one in `Lint.swift`. Can we improve it to follow DRY principle?